test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -2080,6 +2193,352 @@ def test_cgroup_use_hierarchy(self):
         else:
             self.assertFalse(1, "File %s not present" % fpath)
 
+    def check_req_rjs(self):
+        """
+        Check the requirements for the reliable job startup tests.
+        Return 1 if requirements are not satisfied.
```

Comment:
So the requirement we're checking is that hostA has two vnodes and hostB has one, correct? You should add that to the comment.

Reply:
You are correct - I have added that in the comment.